src/Modules/Reports/Commands.Reports.Test/CopyPowerBIReportTests.cs

```diff
@@ -130,15 +130,15 @@ public void CopyPowerBIReportObjectTest()
             var expectedResponse = new Report();
             var client = new Mock<IPowerBIApiClient>();
             client.Setup(x => x.Reports
-                .CopyReport(reportName, Guid.Empty.ToString(), reportId.ToString(), Guid.Empty.ToString(), Guid.Empty.ToString()))
+                .CopyReport(reportName, Guid.Empty, reportId, Guid.Empty, Guid.Empty))
```

Comment:
I know the original code did this, but we should avoid using blank/ambiguous values like this for our tests.